Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -651,111 +577,112 @@ string TryToFindLogo(string uri, string path)
-                    return new BitmapImage(new Uri(Constant.MissingImgIcon));
-                }
-            }
-
-            private ImageSource PlatedImage(BitmapImage image)
-            {
-                if (!string.IsNullOrEmpty(BackgroundColor) && BackgroundColor != "transparent")
-                {
-                    var width = image.Width;
-                    var height = image.Height;
-                    var x = 0;
-                    var y = 0;
-
-                    var group = new DrawingGroup();
-
-                    var converted = ColorConverter.ConvertFromString(BackgroundColor);
-                    if (converted != null)
-                    {
-                        var color = (Color)converted;
-                        var brush = new SolidColorBrush(color);
-                        var pen = new Pen(brush, 1);
-                        var backgroundArea = new Rect(0, 0, width, width);
-                        var rectabgle = new RectangleGeometry(backgroundArea);
-                        var rectDrawing = new GeometryDrawing(brush, pen, rectabgle);
-                        group.Children.Add(rectDrawing);
-
-                        var imageArea = new Rect(x, y, image.Width, image.Height);
-                        var imageDrawing = new ImageDrawing(image, imageArea);
-                        group.Children.Add(imageDrawing);
-
-                        // http://stackoverflow.com/questions/6676072/get-system-drawing-bitmap-of-a-wpf-area-using-visualbrush
-                        var visual = new DrawingVisual();
-                        var context = visual.RenderOpen();
-                        context.DrawDrawing(group);
-                        context.Close();
-                        const int dpiScale100 = 96;
-                        var bitmap = new RenderTargetBitmap(
-                            Convert.ToInt32(width), Convert.ToInt32(height),
-                            dpiScale100, dpiScale100,
-                            PixelFormats.Pbgra32
-                        );
-                        bitmap.Render(visual);
-                        return bitmap;
-                    }
-                    else
-                    {
-                        ProgramLogger.LogException($"|UWP|PlatedImage|{Package.Location}" +
-                                                   $"|Unable to convert background string {BackgroundColor} " +
-                                                   $"to color for {Package.Location}", new InvalidOperationException());
-
-                        return new BitmapImage(new Uri(Constant.MissingImgIcon));
-                    }
-                }
-                else
-                {
-                    // todo use windows theme as background
-                    return image;
-                }
-            }
-
+            #region logo legacy 
+            // preserve for potential future use
```

Comment:
what can we use this for in the future

Reply:
this part fills the background of the icon with background color in appmanifest.Technically background + color is the full icon that app developer want to show. onesounds removed it in preview pr. I move it here to avoid conflict. Still I feel like maybe we may need to look into it later so I just commented it.

Reply:
Ok sounds good